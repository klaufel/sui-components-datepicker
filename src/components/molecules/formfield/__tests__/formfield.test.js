import React from 'react'
import {render} from '@testing-library/react'
import FormField from '..'

describe('FormField', () => {
  test('renders the component', () => {
    const {container} = render(<FormField label="Default label" name="input" />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
