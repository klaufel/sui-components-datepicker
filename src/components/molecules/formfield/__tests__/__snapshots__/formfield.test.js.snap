// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormField renders the component 1`] = `
<div
  class="FormField"
>
  <label
    class="Label"
    for="input"
  >
    Default label
  </label>
  <input
    class="Input"
    id="input"
    type="text"
    value=""
  />
</div>
`;
